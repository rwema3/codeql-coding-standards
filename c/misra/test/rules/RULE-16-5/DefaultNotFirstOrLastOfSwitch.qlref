rules/RULE-16-5/DefaultNotFirstOrLastOfSwitch.ql