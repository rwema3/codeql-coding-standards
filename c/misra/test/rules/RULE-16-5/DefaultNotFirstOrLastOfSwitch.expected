| test.c:16:3:16:10 | default:  | $@ statement does not have $@ case as first or last switch label. | test.c:12:3:22:3 | switch (...) ...  | Switch | test.c:16:3:16:10 | default:  | default |
